README.md

```diff
@@ -0,0 +1,37 @@
+# covid-sequence-analysis-workflow
+
+This is the official repository of the SARS-CoV-2 variant surveillance pipeline developed under the [Versatile Emerging infectious disease Observatory (VEO)](https://www.globalsurveillance.eu/projects/veo-versatile-emerging-infectious-disease-observatory) project. The project consists of 20 European partners. It is funded by the European Commission.
+
```

Comment:
Maybe can also mention that the pipeline has been integrated on EMBL-EBI infrastructure to automatically process raw SARS-CoV-2 read data, presenting in the COVID-19 Data Portal: https://www.covid19dataportal.org/sequences?db=sra-analysis-covid19&size=15&crossReferencesOption=all#search-content.